test/lib/formatter/RuleResultRecombinator.test.ts

```diff
@@ -342,7 +342,7 @@ describe('RuleResultRecombinator', () => {
 
 				rule = driver['rules'][1];
 				expect(rule.id).to.equal('ApexDoc');
-				expect(rule.shortDescription.text).to.satisfy(r => r.startsWith('This rule validates that:\n\n* ApexDoc comments'));
```

Comment:
The docs were changed [by this commit in PMD](https://github.com/pmd/pmd/commit/a0b0be35bbe5b861c8f8607671d3fde2e6efd39c#diff-dc3cca7a503e178c3f3a42c2fef5388506599f6598143d51014867a06a57df64) which broke the test. I changed the test to be a little more forgiving.